pkg/kubelet/prober/prober_manager.go

```diff
@@ -257,6 +257,8 @@ func (m *manager) UpdatePodStatus(podUID types.UID, podStatus *v1.PodStatus) {
 				ready = !exists
 			}
 			podStatus.ContainerStatuses[i].Ready = ready
+		} else {
+			podStatus.ContainerStatuses[i].Ready = false
```

Comment:
Can we add a testcase for this in TestUpdatePodStatus?

Reply:
I have add a testcase, and delete the unreachable code